cart/cart.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="cart.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Tenor+Sans&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <button class="btn-style">
+          <img
+            src="../assets/general-assets/close.svg"
+            alt="close"
+            width="35px"
+          />
+        </button>
+        <h2>CART</h2>
+      </nav>
+    
+      <main>
+        <section>
+          <div class="img-container">
+            <img src="../assets/cart-page/cart1.png" alt="NOTFOUND" />
+          </div>
+          <div class="product-info">
+            <h3>LAMEREI</h3>
+            <p>Recycle Bouncle Knit Cardigan Pink</p>
+            <div class="quantity">
+              <button class="btn-style">
+                <img src="../assets/general-assets/minus.svg" alt="close" />
+              </button>
+              <p class="counter">1</p>
+              <button class="btn-style">
+                <img src="../assets/general-assets/Plus.svg" alt="close" />
+              </button>
+            </div>
+            <p class="pricing">$120</p>
+          </div>
+        </section>
+
+        <section>
+            <div class="img-container">
+              <img src="../assets/cart-page/cart2.png" alt="NOTFOUND" />
+            </div>
+            <div class="product-info">
+              <h3>5252 BY O!OI</h3>
+              <p>2021 Signature Sweatshirt [NAVY]</p>
+              <div class="quantity">
+                <button class="btn-style">
+                  <img src="../assets/general-assets/minus.svg" alt="close" />
+                </button>
+                <p class="counter">1</p>
+                <button class="btn-style">
+                  <img src="../assets/general-assets/Plus.svg" alt="close" />
+                </button>
+              </div>
+              <p class="pricing">$120</p>
+            </div>
+          </section>
+      </main>
+    </header>
```

Comment:
move the closing tag of the header after the nav closing tag in line 28

https://github.com/GSG-CF05/FashionApp-team3/blob/a0617862993b487dae34a3892cf3d0ed785a20c6/cart/cart.html#L26-L28